README.md

```diff
@@ -1,48 +1,248 @@
+
+   Run the migrate command to apply any necessary database schema changes:
+   
+```bash
 cd mesa_gui
 python manage.py migrate
+```
+
+5. **Create an administrative user**
+
+  Create a superuser account for accessing the web application:
+
+```bash
 python manage.py createsuperuser
+```
+
+6. **Load MESA jobs data**
+
+   Load the MESA jobs data into the Django database. These fixtures define the job functions that the MESA-GUI will use for scanning activities:
+
+```bash
 python manage.py loaddata mesa/fixtures/mesajobs.json
+```
 
-# generate a self-signed SSL certificate
+7. **Generate a self-signed SSL certificate**
+
+  Generate a self-signed SSL certificate to securely host the MESA-GUI over HTTPS, ensuring encrypted communication:
+
+```bash
 openssl req -x509 -newkey rsa:4096 -keyout key.pem -out cert.pem -days 365 -nodes -subj "/C=/ST=/L=/O=/OU=/CN="
+```
+
+8. **Start the web application**
+
+   Start the web application using the SSL certificate:
 
-# start the web application
+```bash
 python manage.py runsslserver --certificate cert.pem --key key.pem 0.0.0.0:8080
 ```
 
-# Migrations
-Migrations will need to be created when making alterations to the SQL database. The following commands can be used to create a migration:
+## Navigating the MESA-GUI
+
+Once the installation process is complete, the MESA-GUI web interface can be accessed by browsing to the IP address of the machine hosting the web server:
+
+```
+https://[IP Address]:8080
+```
+![image](https://github.com/user-attachments/assets/d804ab89-2d21-467a-b80e-7bf3082ea614)
+
+Login to the application using the credentials created during the installation process to access the MESA dashboard.
+
+![image](https://github.com/user-attachments/assets/e5a0d4f8-2dc7-4448-99e6-788c0d08e8b6)
+
+Configuring Scan Settings
+--------------
+Prior to running any scans, browse to the settings tab to configure the scan scope and pertinent deliverable variables.
+
+![image](https://github.com/user-attachments/assets/3b13e3bd-ab9e-4fae-b5e4-3d2e28b5af61)
+
+Update the following variables in the settings tab:
+
+| Variable | Description | Default | Required |
+|----------|-------------|---------|---------|
+| Project Name | The name used to track the project | RV#### | yes |
+| Customer Name | The customer full name as it will appear on the deliverables | Acme Corporation | yes |
+| Customer Initials | The customer shortname as it will appear on the deliverables | ACME | yes |
+| In-Scope Systems | The individual IP addresses or cidr blocks to be included in scanning activity | none | yes |
+| Excluded Systems | The individual IP addresses or cidr blocks to be excluded from scanning activity | none | no |
+
+MESA Job Definitions
+--------------
+Once the settings have been configured browse to the dashboard tab to begin running the scans.
+
+![image](https://github.com/user-attachments/assets/1091208a-12a2-463e-81aa-960d47dc9497)
+
+| Job Title | Description | Required | 
```

Comment:
might be worth mentioning (just above this) what required means in this context

Reply:
I added a note to this section.